hyperspy/_signals/signal1d.py

```diff
@@ -1068,75 +1069,24 @@ def remove_background(
             signal_range='interactive',
             background_type='Power Law',
             polynomial_order=2,
+            snip_width=10,
+            snip_iterations=16,
             fast=True,
             zero_fill=False,
             plot_remainder=True,
             show_progressbar=None, display=True, toolkit=None):
-        """
```

Comment:
It looks like you accidentally removed the docstring from the `remove_background` method. When you add it back, you could also add "Snip" to the background_type as well as document the snip_width and snip_iterations.

Reply:
The disappearance of the docstring is also what is causing the tests to fail.